pkg/mdformatter/linktransformer/link.go

```diff
@@ -180,6 +190,20 @@ func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []
 	v.c.OnError(func(response *colly.Response, err error) {
 		v.rMu.Lock()
 		defer v.rMu.Unlock()
+		// Colly stores requests in cache, even when they have errors.
+		// So need to delete requests with errors from cache database.
+		if v.storage != nil {
+			h := fnv.New64a()
```

Comment:
Let's add that to separate method, ideally on storage side? (hashing request).

Are you sure it's the same as colly hash. What if they change that next version (quite prone to errro code)

Reply:
I wanted to do it on storage side too, but have to conform colly storage interface so that we can pass that in.